docs/scripting/removed Callbacks/OnPlayerPrivmsg.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
I think just putting this back into the callbacks directory would be fine. I don't see any more callbacks being added so there's probably no need for a whole directory for it.

Reply:
I think, may the better idea be to create a full section of removed functions/callbacks, because putting a removed callback back into callbacks folder makes no sense to me, because it's no longer used, and, thus, why does it have to be documented under callbacks section?

Reply:
But there are only two removed callbacks by my count, and it might not even be worth documenting them, these were removed like 10 years ago.

Reply:
Right, I will then delete removed callbacks section

Reply:
Okay, I think I removed removed Callbacks folder and it from footer in docusaurus.config